portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/masterspagetemplate/MastersPageTemplates.testcase

```diff
@@ -1710,4 +1710,50 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-124104. View the page editor is available with custom master page after staging is enable."
```

Comment:
The page editor of content page based on custom master should be available in staging site.